test/test_line_break.py

```diff
@@ -0,0 +1,95 @@
+from fpdf import FPDF
+from fpdf.line_break import Fragment, MultiLineBreak, text_line
+
+
+def test_one_line():
+    pdf = FPDF()
+    pdf.set_font("Times", size=32)
+    fragments = [
+        Fragment.from_string("hello world", "", False),
+    ]
+    multi_line_break = MultiLineBreak(
+        fragments, pdf.get_normalized_string_width_with_style
+    )
+    assert multi_line_break.get_line_of_given_width(100000) == text_line(
+        fragments, 4583, 1, False
```

Comment:
Would you mind specifying constructor parameters names here, for extra readibility?